tests/product/base_product_case.py

```diff
@@ -398,6 +398,11 @@ def assert_installed(self, container, cluster=None, msg=None):
                 check_rpm, RPM_BASENAME + '\n', msg=msg
             )
         except OSError as e:
+            if isinstance(cluster, DockerCluster):
```

Comment:
Is this just a temporary change?  I'm worried about cluttering up the system running the tests and forgetting about these images lying around